lib/protocols/vast/KalturaVastParser.js

```diff
@@ -9,7 +9,14 @@ class KalturaVastParser
 	static parse(vastUrl, headers, timeout, callback)
 	{
 		const customizedHeaders = KalturaVastParser._custoimizeHeaders(headers);
-		vastClient.get(vastUrl, customizedHeaders, timeout, callback);
+		vastClient.get(vastUrl, customizedHeaders, timeout,
+			function (vastObject)
+			{
+				if (KalturaVastParser._validate(vastObject))
+					callback(vastObject);
+				else
+					callback({ads:[], errorURLTemplates:[]});
```

Comment:
Please add a print here that says we encountered an error and some text , use KalturaLogger.error